lib/raidenclient/RaidenClient.ts

```diff
@@ -70,8 +70,8 @@ type ChannelEvent = {
 };
 
 interface RaidenClient {
-  on(event: 'swap', listener: (order: Order) => void): this;
-  emit(event: 'swap', order: Order): boolean;
+  on(event: 'connectionVerified', listener: (newAddress?: string) => void): this;
```

Comment:
I'm wondering if these should also be moved to `BaseClient`?

Reply:
Yeah I was on the fence but in thinking about it some more it makes sense, I can call the parameter `newIdentifier` to be generic. I'll make that change.